core/deployment/src/main/java/io/quarkus/deployment/pkg/PackageConfig.java

```diff
@@ -202,6 +202,13 @@ interface AppcdsConfig {
              */
             @WithDefault("true")
             boolean useContainer();
+
+            /**
+             * Whether to use <a href="https://openjdk.org/jeps/483">Ahead-of-Time Class Loading & Linking</a> introduced in JDK
+             * 24.
+             */
+            @WithDefault("false")
+            boolean useAot();
```

Comment:
The enum `JvmStartupOptimizerArchiveType.AOT` is spelled in uppercase, as it appears to be an abbreviation, which is commonly written in uppercase in this PR. Would it be considerable to apply the same naming convention here for consistency and clear communication?
```suggestion
            boolean useAOT();
```
<img width="1113" alt="image" src="https://github.com/user-attachments/assets/8d524b85-a86e-4ebd-9f91-3b8ad1755be2" />

https://de.wikipedia.org/wiki/Ahead-of-time-Compiler

Reply:
We don't want to do that as it changes the name of the property users have to specify